openj9.test.sharedClasses.jvmti/src/native/Makefile

```diff
@@ -66,7 +66,7 @@ P=:
 ###
 # Check platform is set to a single valid value
 ###
-VALID_PLATFORMS?=aix_ppc-32,aix_ppc-64,linux_390-31,linux_390-64,linux_ppc-32,linux_ppc-64,linux_x86-32,linux_x86-64,win_x86-32,win_x86-64,zos_390-31,zos_390-64,linux_ppcle-64,linux_arm-32,linux_aarch64-64,osx_x86-64
+VALID_PLATFORMS?=aix_ppc-32,aix_ppc-64,linux_390-31,linux_390-64,linux_ppc-32,linux_ppc-64,linux_x86-32,linux_x86-64,win_x86-32,win_x86-64,zos_390-31,zos_390-64,linux_ppcle-64,linux_arm-32,linux_aarch64-64,osx_x86-64,linux_arm-64
```

Comment:
Is linux_aarch64-64 unused then?  Or do we have a situation where there are multiple names for same things in different places?  I am fine to merge this, but want to check if we can remove the other, or if both are currently used.

I see that this addresses this failure: https://ci.adoptopenjdk.net/view/Test_system/job/Test_openjdk11_j9_extended.system_aarch64_linux/48/console

Reply:
`java_platform` is returning `linux_arm-64` - which is set in top.xml by using Java System properties.  

As in https://ci.adoptopenjdk.net/view/Test_system/job/Test_openjdk11_j9_extended.system_aarch64_linux/48/consoleFull

```18:47:08       [echo] make -C /home/jenkins/workspace/Test_openjdk11_j9_extended.system_aarch64_linux/openjdk-tests/system/openj9-systemtest/openj9.test.sharedClasses.jvmti/src/native SRCDIR=/home/jenkins/workspace/Test_openjdk11_j9_extended.system_aarch64_linux/openjdk-tests/system/openj9-systemtest/openj9.test.sharedClasses.jvmti/src/native **PLATFORM=linux_arm-64** 
```

I do not see `java_platform` being set to `linux_aarch64-64` in any aarch64 builds (checked jdk 8, 11, 13 system test builds at adopt). 

It appears Java system properties was previously returning  `linux_aarch64-64` as the value for platform and now switched to  `linux_arm-64` ? I am not sure.

Reply:
A test should be to remove `linux_aarch64-64` - and run grinders in jdk 8, 11, 13 on aarch64 builds and make sure it compiles

Reply:
Also, it's noteworthy that `linux_aarch64-64` doesn't exist in the modularity test's native compile script: https://github.com/AdoptOpenJDK/openjdk-systemtest/blob/master/openjdk.test.modularity/src/tests/com.test.jlink/native/makefile 

We have `linux_arm-64` there - and it compiles fine in JDK11 and 13 on both Adopt and OpenJ9 JDKs.